Matter: add device information (#31267)
home-assistant/home-assistant.github.io: source/_integrations/matter.markdown

   - For example, if you shared it with Google Home, it lists **Google LLC**.
     ![image](/images/integrations/matter/matter_share-device-with-other-platform.png)

## About Matter device information

> The device information section provides some networking information of a device. For devices that are behind a bridge, this information is not available.

Reviewer: ```suggestion
The device information section provides some diagnostic information of a device.
```

I'd move this more to the back: For devices that are behind a bridge, this information is not available.

Author: Well, if it is stated here, I know that I can stop reading if I have a bridged device

1. To view the device details, go to {% my integrations title="**Settings** > **Devices & services**" %} and on the **Matter (BETA)** integration card, select **Devices**.
2. From the list of devices, select the device you want to look at.

<p class='img'>
<img src='/images/integrations/matter/matter_device-info.png' alt='Matter device information'>
</p>

This section provides a bit more information on some of the categories:

**Network type**: Shows which network type the device belongs to. For example, Thread or Wi-Fi.

**Device type**: There are 4 device types:

> - **End device**: End device that always keeps it radio on. Typically these devices are not battery-powered.

Reviewer: ```suggestion
- **End device**: End device that always keeps its radio on. Typically these devices are not battery-powered.
```

> - **Sleepy end device**: non-routing end device. Normally disabled, wakes on occasion to poll for messages from its parent. Typically battery-powered.

Reviewer: ```suggestion
- **Sleepy end device**: non-routing end device. Normally sleeping, wakes on occasion to check for messages. Typically battery-powered.
```

> - **Routing end device**: End device that always keeps it radio on and can route traffic for other, non-routing end devices.

Reviewer: ```suggestion
- **Routing end device**: End device that always keeps its radio on and can route traffic for other, routing and non-routing end devices.
```

Reviewer: This device can also be seen as a range extender. It extends the mesh

- **Bridge**

> **Network name**: Name of the network the device joined when it was first commissioned first.

Reviewer: ```suggestion
**Network name**: Name of the network the device joined when it was commissioned.
```

> **IP addresses**: For Thread networks, typically more than one IPv6 address is shown: link local, unique local, and global unicast. Sometimes, no IPv6 address is shown.

Reviewer: ```suggestion
**IP addresses**: Typically more than one IPv6 address is shown: link local, unique local, and global unicast. In some cases a device also supports IPv4. In that case there will also be listed an IPv4 address here.
```

the missing IP address on the P2 sensor is a bug actually

Reviewer: so you want to explain all the found ip addresses ? You could also shortcut it to: the IP adresses reported/used by this device.

Author: I added it like this because it makes it more clear why there are multiple addresses, and that they are actually just versions of. I didn't look at the address closely. So my first guess was that the first address was its own, and then a list of the next neighbors or something.
